internal/model/todos.go

```diff
@@ -10,43 +14,42 @@ type Todo struct {
 
 // TodoList kind of a has many struct
 type TodoList struct {
-	ID    string `json:"id"`
-	Name  string `json:"name"`
-	Todos []Todo `json:"todos"`
+	ID    string  `json:"id"`
+	Name  string  `json:"name"`
+	Todos []*Todo `json:"todos"`
 }
 
 // GetTodosListByID returns slice of Todos based on the listID passed in
-func GetTodosListByID(listID string) (TodoList, error) {
+func GetTodosListByID(listID string) (*TodoList, error) {
```

Comment:
Here we're saying that we return a pointer to a TodoList. We'll create the TodoList then tell you where to find it. Instead of creating it ourselves then just making a copy to give them. In order for us to send them the one we created we need to point to it...